packages/blade/src/components/Input/PhoneNumberInput/CountrySelector.web.tsx

```diff
@@ -52,6 +52,7 @@ const CountrySelector = ({
   size,
 }: CounterSelectorProps): React.ReactElement => {
   const [isDropdownOpen, setIsDropdownOpen] = React.useState(false);
+  const [isHoverOrFocus, setIsHoverOrFocus] = React.useState(false);
```

Comment:
Do we have to do this in all dropdown buttons? (searchinput, textinput, etc)

Reply:
no